ipetcare.spa/src/state/examinationParameters/examinationParametersActions.ts

```diff
@@ -0,0 +1,240 @@
+import { history } from './../../index'
+import { ExaminationParameters as examinationParameters } from '../../api'
+import { ThunkAction } from 'redux-thunk'
+import { Dispatch } from 'redux'
+import { RootState, RootActions } from '../store'
+import { ExaminationParameter } from './examinationParametersReducer'
+
+export type ThunkResult<R> = ThunkAction<R, RootState, undefined, RootActions>
+export enum ExaminationParametersActionParameters {
+    GET_EXAMINATION_PARAMETERS = 'GET_EXAMINATION_PARAMETERS',
+    GET_EXAMINATION_PARAMETERS_SUCCESS = 'GET_EXAMINATION_PARAMETERS_SUCCESS',
+    GET_EXAMINATION_PARAMETERS_FAIL = 'GET_EXAMINATION_PARAMETERS_FAIL',
+    GET_EXAMINATION_PARAMETER = 'GET_EXAMINATION_PARAMETER',
+    GET_EXAMINATION_PARAMETER_SUCCESS = 'GET_EXAMINATION_PARAMETER_SUCCESS',
+    GET_EXAMINATION_PARAMETER_FAIL = 'GET_EXAMINATION_PARAMETER_FAIL',
+    CREATE_EXAMINATION_PARAMETER = 'CREATE_EXAMINATION_PARAMETER',
+    CREATE_EXAMINATION_PARAMETER_SUCCESS = 'CREATE_EXAMINATION_PARAMETER_SUCCESS',
+    CREATE_EXAMINATION_PARAMETER_FAIL = 'CREATE_EXAMINATION_PARAMETER_FAIL',
+    UPDATE_EXAMINATION_PARAMETER = 'UPDATE_EXAMINATION_PARAMETER',
+    UPDATE_EXAMINATION_PARAMETER_SUCCESS = 'UPDATE_EXAMINATION_PARAMETER_SUCCESS',
+    UPDATE_EXAMINATION_PARAMETER_FAIL = 'UPDATE_EXAMINATION_PARAMETER_FAIL',
+    DELETE_EXAMINATION_PARAMETER = 'DELETE_EXAMINATION_PARAMETER',
+    DELETE_EXAMINATION_PARAMETER_SUCCESS = 'DELETE_EXAMINATION_PARAMETER_SUCCESS',
+    DELETE_EXAMINATION_PARAMETER_FAIL = 'DELETE_EXAMINATION_PARAMETER_FAIL',
+}
+
+// FETCH EXAMINATION_PARAMETER LIST
+interface GetExaminationParameters {
+    type: ExaminationParametersActionParameters.GET_EXAMINATION_PARAMETERS
+}
+
+interface GetExaminationParametersSuccess {
+    type: ExaminationParametersActionParameters.GET_EXAMINATION_PARAMETERS_SUCCESS
+    payload: any
```

Comment:
Czy payload tutaj i w innych przypadkach powinien być typu any? Zdaje się, że na discordzie chyba o tym rozmawialiśmy i udało Ci się to poprawić :D